userservice/src/main/java/com/infodation/userservice/models/UserDTO.java

```diff
@@ -0,0 +1,57 @@
+package com.infodation.userservice.models;
+
+public class UserDTO {
+    private String userId;
+    private String firstName;
+    private String lastName;
+    private String sex;
+    private String email;
+
+    public UserDTO(String userId, String firstName, String lastName, String sex, String email) {
+        this.userId = userId;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.sex = sex;
+        this.email = email;
+    }
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getSex() {
+        return sex;
+    }
+
+    public void setSex(String sex) {
+        this.sex = sex;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+}
```

Comment:
Use `Lombok` to avoid boring getter setter methods. You can see example at class `CreateUserDTO`